content/kubeone/master/references/kubeone_cluster_v1beta1/_index.en.md

```diff
@@ -4,6 +4,8 @@ date = 2021-02-10T12:00:00+02:00
 weight = 1
 +++
 
+This page describes API documentation (yaml file layout) for the kubeone configuration file (default name kubeone.yaml).
```

Comment:
Please don't edit this page, this a copy of autogenerated API doc in kubeone repository

Reply:
OK.. Let me address this. But if you look at similar documentation from other places like [Kops API documentation](https://kops.sigs.k8s.io/cluster_spec/), the page is far more reader friendly. I believe that if we do not edit it here, we need to probably add some contextualized header in the page where it gets generated in first place. Again that's just my thought.

Consequently, all the updates that I did to the table.. Will that also need to be done elsewhere? e.g. AzureSpec, AWSSpec etc? Could you let me know where can I make upstream updates so that generated page will include these changes

Reply:
As I've said, the doc is autogenerated in kubeone. It uses kubeone v1beta1 API to generate and take comments from there. So if you edit the comments in that API then yes — your changes will end up here, in autogenerated doc.

https://github.com/kubermatic/kubeone/blob/master/hack/apidoc-gen.sh here's the script to generate the doc.

Reply:
I have removed this page change.. as requested. Please check